scripts/TerminateNotStartedAssessment.php

```diff
@@ -86,10 +86,16 @@ public function __invoke($params)
                     $data[DeliveryMonitoringService::DELIVERY_EXECUTION_ID]
                 );
                 if ($testSessionService->isExpired($deliveryExecution)) {
-                    $deliveryExecutionStateService->cancelExecution($deliveryExecution, [
+                    $cancel = $deliveryExecutionStateService->cancelExecution($deliveryExecution, [
```

Comment:
Better check if delivery execution is cancelable (`\oat\taoProctoring\model\implementation\DeliveryExecutionStateService::isCancelable`) and if so, cancel it or pause otherwise.